src/volatile_memory.rs

```diff
@@ -588,6 +588,10 @@ impl<B: BitmapSlice> Bytes<usize> for VolatileSlice<'_, B> {
     /// assert_eq!(res.unwrap(), 4);
     /// ```
     fn write(&self, buf: &[u8], addr: usize) -> Result<usize> {
+        if buf.len() == 0 {
```

Comment:
I think we are already returning Ok(0) in this case, but I'll double-check.

Reply:
This seems to be the case, so I think we can keep only the tests here.

Reply:
This is not the case when the volatile slice is empty. I've added a test case for both use cases:
- volatile slice is empty, read/write size is 0
- volatile slice is not empty, read/write size is 0